pom.xml

```diff
@@ -22,6 +22,11 @@
             <artifactId>jersey-client</artifactId>
             <version>2.25</version>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
```

Comment:
You are not writing enterprise project, why do not use latest Junit as well as try java 9 ?